seq2science/rules/qc.smk

```diff
@@ -751,7 +751,7 @@ rule multiqc_schema:
         order = -954
         deseq2_imgs = ""
         deseq2_order = ""
-        for contrast in expand_contrasts(samples, config):
+        for contrast in expand_contrasts(samples, config["contrasts"]):
```

Comment:
where is contrasts being set? Do you mean `DE_contrasts`?

Reply:
`DE_contrasts` contains the file paths to each DESeq2 output.
`expand_contrasts()` returns the actual contrasts, which are parsed from the config.

so this line is correct, however, we could also make it into another global list, or run `config["contrasts"] = expand_contrasts(samples, config["contrasts"])` somewhere in the config. Do you have a preference?